### PR TITLE
adding Dapr Repository Index for contributing

### LINKED ISSUE #4076: Enhancement: Add Index of Dapr Repositories to Contributing Overview Page
**What content needs to be created or modified?**

The Contribution Overview page requires the addition of an index detailing Dapr repositories. This index will assist contributors in identifying areas where they can make contributions effectively within the Dapr project.

**Describe the solution you'd like**

I propose adding an index list of Dapr repositories to facilitate users interested in contributing. Each repository link should direct users to the CONTRIBUTING.md page of the respective repository. This index would streamline the process of finding specific areas for contribution within the Dapr organization.
**Where should the new material be placed?**

**Additional context**

It's like giving contributors a roadmap to the heart of Dapr's world. By adding an index of Dapr repositories to the Contribution Overview page, we're making it super easy for everyone to find where they fit in and contribute.

### PR DESCRIPTION
**Pull request by AnnuCode**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Added an index list of Dapr repositories to facilitate users interested in contributing.
## Issue reference

Fixes #4076

### CODE REVIEW

File: daprdocs/content/en/contributing/contributing-overview.md [hunk 1]
```diff
@@ -8,11 +8,23 @@ description: >
 ---
 
 Thank you for your interest in Dapr!
-This document provides the guidelines for how to contribute to the [Dapr project](https://github.com/dapr) through issues and pull-requests. Contributions can also come in additional ways such as engaging with the community in community calls, commenting on issues or pull requests and more.
+This document provides the guidelines for how to contribute to the [Dapr project](https://github.com/dapr) through issues and pull requests. Contributions can also come in additional ways such as engaging with the community in community calls, commenting on issues or pull requests, and more.
 
 See the [Dapr community repository](https://github.com/dapr/community) for more information on community engagement and community membership.
 
-> If you are looking to contribute to the Dapr docs, please also see the specific guidelines for [docs contributions]({{< ref contributing-docs >}}).
+## Dapr Repository Index
```
Comment (marcduiker) [inline daprdocs/content/en/contributing/contributing-overview.md:15]: Can you add dapr/cli to this list as well? Then we have all the major repositories covered.

**Review by marcduiker — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/contributing/contributing-overview.md (hunk 1 above)
Comment (AnnuCode) [reply daprdocs/content/en/contributing/contributing-overview.md:15]: added the cli repo ✅

**Review by marcduiker — APPROVED** [review]
LGTM

**Review by msfussell — APPROVED** [review]

**Comment by marcduiker** [comment]
@holopin-bot @AnnuCode Thanks Annu! 😁

**Pull request merged**